cell_types_utils.R

```diff
@@ -42,7 +49,7 @@ get_shared_terms_prop = function(reference_labs, pred_labs, trivial_terms){
 
 ### find per-class/median F-1 scores and accuracy ###
 # adapted from https://github.com/tabdelaal/scRNAseq_Benchmark/blob/master/evaluate.R
```

Comment:
Could you state clearly in the README that the scRNAseq_Benchmark project was an inspiration please, and state which methods are borrowed from that work?